deploy/src/main/deploy/helm/values.yaml

```diff
@@ -27,6 +31,11 @@ healthCheckPort: ${vertx.health.port}
 # this flag is used to determine whether we need to deploy Route resources as well
 openshift: false
 
+# whether we are deploying to Azure (true) or plain Kubernetes (false)
+# do not use with "openshift: true".
+# this flag is used to determine whether we need to deploy service type "LoadBalancer" for external services
+azure: false
```

Comment:
I agree. Let's change this to describe the feature. We can also change "openshift" property to something like "platform", so that we can have

platform: kubernetes (default)
platform: openshift
platform: azure (if needed)

BTW. I'm also using "Kubernetes for Docker" and it supports LoadBalancer nicely. I'm not sure about Minikube, but maybe we can make that default deployment option?

Reply:
Other than this, it looks great. Thanks for the contribution

Reply:
Hi @dejanb, @sophokles73 
Thanks for a feedback. Yep, I think it will be better to have more generic parameter name (like @sophokles73 proposed: "useLoadBalancer").
@dejanb, regarding your proposal to change "openshift: false" to "platform: kubernetes":
It means, that on each "*route.yaml" file we need to change "{{if eq .Values.openshift true}}" to "{{if eq .Values.platform "openshift"}}". 
I can do this (if you are OK).

Thank you!

Reply:
@spoplavskiy Yeah, sounds great. I can test it here then to make sure there's no regressions.

Reply:
Added commit: 4ad66dcdde675acb6ce7af13a3133f93bd4ca8fc